public/app/percona/inventory/Tabs/Nodes.utils.ts

```diff
@@ -0,0 +1,12 @@
+export const stripNodeId = (nodeId: string) => {
```

Comment:
isn't this function repeated?

Reply:
Yes it is same as in services. But with a little change. Instead of service_id there is node_id in regex. I can change it to something like stripId(id: string, type: 'node_id' | 'service_id') but not sure if it is necessary.

Reply:
I mean this function is repeated at "public/app/percona/check/components/FailedChecksTab/FailedChecksTab.utils.ts"

Reply:
True, good catch. Removed.